src/main/java/CashDrawer.java

```diff
@@ -86,6 +86,14 @@ public void takeBillsFromRegister(int twenty, int ten, int five, int two, int on
         this.one -= one;
     }
 
+    private boolean getAmountOfEachTotal(int changeAmount){
+        return false;
+    }
+
+    public void makeChange(int changeAmount){
```

Comment:
Remember what we asked before about void methods? How would you make this testable?